Makefile

```diff
@@ -1,9 +0,0 @@
-prettify:
-	black .
-	isort .
-
-lint:
-	flake8 .
-	black --check .
-	isort -c .
-	mypy --strict .
```

Comment:
could redefine as `pre-commit run --all-files` but I mean, not a must

Reply:
It's not exactly the same though, as the pre-commit things will change files...

Reply:
yes, there is a slight difference, but I do prefer running it on everything if I do it manually while developing

but that is ofc only a personal preference :person_bald: 

we can have it without the `--all-files` too

Reply:
I have an alias `pcr`, which does `pre-commit run --from-ref origin/$(git kit main-branch) --to-ref HEAD` (i.e. fixes up the branch I'm working on) (and could have `pca` for `--all-files`)...

Reply:
those are definitely handy if you've got slower lints ! 🤩

Reply:
no need to add the `make lint`, the README mention is good enough